waspc/src/Wasp/Psl/Parser/Schema.hs

```diff
@@ -32,6 +34,8 @@ schema = do
     many $
       choice
         [ Psl.Schema.ModelBlock <$> model,
+          Psl.Schema.ViewBlock <$> view,
+          Psl.Schema.TypeBlock <$> typeBlock,
```

Comment:
Why is `typeBlock` only parser that has `Block` suffix here?

Reply:
I wouldn't define a function with a name `type` since it's a reserved Haskell keyword.